logic/dapp.js

```diff
@@ -186,12 +186,14 @@ DApp.prototype.verify = function(transaction, sender, cb, tx) {
 		}
 	}
 
-	return (tx || library.db).dapps
-		.getExisting({
+	return library.storage.entities.Transaction.getExistingDapps(
+		{
 			name: transaction.asset.dapp.name,
 			link: transaction.asset.dapp.link || null,
-			transactionId: transaction.id,
-		})
+			transactionId_ne: transaction.id,
+		},
```

Comment:
BTW you are passing `transactionId_ne` here and using `transactionId` in the function.

Reply:
This have been already fixed by same comment by Lucas. but will replace whole method so resolving.